study.md

```diff
@@ -40,5 +42,5 @@ In your own words explain what a prototype is.  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+Every function has a protype property, when you want to implement attributes stored within the original to a newly created object you use __proto__ to do this, so all attributes and properties flow down to the new object.  This allows you to reuse your work over and over again.  The new object can have its own attributes also so this allows for the creation of more complex objects with less work. I used the required reading and a conversation with a seasoned dev. for a little extra clarity.
```

Comment:
This is a good explanation but be careful with using the `.__proto__` property. You would normally attach the methods you want to the prototype of the function that is creating the objects.